spring-data-cassandra/src/test/java/org/springframework/data/cassandra/repository/support/BasicMapIdUnitTests.java

```diff
@@ -41,4 +55,471 @@ public void testMapConstructor() {
 		assertThat(map.get("field1")).isEqualTo(basicMapId.get("field1"));
 		assertThat(map.get("field2")).isEqualTo(basicMapId.get("field2"));
 	}
+
+
+	@Test
+	public void testOne() throws Exception {
+
+		BasicMapId basicMapId = new BasicMapId();
+		basicMapId.putAll(basicMapId);
```

Comment:
Verifies if the putAll method accepts itself as input parameter.
And furthermore verifies if the input parameter is allowed to be empty internally or not.